scripts/get_token.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# This script will connect to the vCD using username and password,
+# and show the header that cointains an authorization token.
+#
+user=$1
+password=$2
+org=$3
+IP=$4
+
+if [ -z "$IP" ]
+then
+    echo "Syntax $0 user password organization hostname_or_IP_address"
+    exit 1
+fi
+
+auth=$(echo -n "$user@$org:$password" |base64)
+
+curl -I -k --header "Accept: application/*;version=27.0" \
```

Comment:
If we're going away to v29 - we could have it in sync I guess

Reply:
I recommend to also to change to 
```suggestion
curl -I -k --header "Accept: application/*;version=29.0" \
```

Reply:
Changed in in both PRs, although the result would be the same.

Reply:
That's the very good.